part1/color.rb

```diff
@@ -0,0 +1,26 @@
+require 'pry'
+
+class Color
+  attr_reader :r, :g, :b
```

Comment:
is you initializer(constructor) receive more than two params use optional parameter 

```
def initialize(params = {})
....
```